### PR TITLE
LPS-150753 | master

### PR DESCRIPTION
**Pull request by gaborlovasdotcom**
[LPS-150753](https://issues.liferay.com/browse/LPS-150753)

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiePreferences.testcase [hunk 1]
```diff
@@ -0,0 +1,153 @@
+@component-name = "portal-security"
+definition {
+
+	property custom.properties = "feature.flag.LPS-142518=true";
+	property portal.upstream = "true";
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiePreferences.testcase:5]: Don't forget to run cases on release run. **property portal.release = "true";**
Meanwhile need to quarantine those blocked cases on release run after add the release property.

Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiePreferences.testcase:6]: Thank you, I added this line.

**Comment by gaborlovasdotcom** [comment]
@JoyceWang08211 I renamed some tests and added missing ones

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**